arches/app/views/base.py

```diff
@@ -96,7 +96,7 @@ def get_context_data(self, **kwargs):
         geom_datatypes = [d.pk for d in models.DDataType.objects.filter(isgeometric=True)]
         geom_nodes = models.Node.objects.filter(
             graph__isresource=True,
-            # graph__publication=True,
+            graph__publication=True,
```

Comment:
same issue here as in advanced_search.py.  we should use the QuerySet API field lookup `isnull`: https://docs.djangoproject.com/en/2.2/ref/models/querysets/#std:fieldlookup-isnull